`prometheus-cpp::util` is not found in prometheus-cpp 1.1.0 and before. Which will make prometheus failed to build

**Describe your environment**

Windows

**Steps to reproduce**

Build otel-cpp with prometheus-cpp 1.1.0

**What is the expected behavior?**
Success

**What is the actual behavior?**

```
[cmake] CMake Error at exporters/prometheus/CMakeLists.txt:33 (target_link_libraries):
[cmake]   Target "opentelemetry_exporter_prometheus" links to:
[cmake] 
[cmake]     prometheus-cpp::util
[cmake] 
[cmake]   but the target was not found.  Possible reasons include:
[cmake] 
[cmake]     * There is a typo in the target name.
[cmake]     * A find_package call is missing for an IMPORTED target.
[cmake]     * An ALIAS target is missing.
[cmake] 
[cmake] 
[cmake] 
```

--- a/exporters/prometheus/CMakeLists.txt
+++ b/exporters/prometheus/CMakeLists.txt
@@ -30,10 +30,12 @@ endif()
 if(TARGET util)
   list(APPEND PROMETHEUS_EXPORTER_TARGETS util)
 endif()
-target_link_libraries(
-  opentelemetry_exporter_prometheus
-  PUBLIC opentelemetry_metrics prometheus-cpp::pull prometheus-cpp::core
-         prometheus-cpp::util)
+set(PROMETHEUS_CPP_TARGETS prometheus-cpp::pull prometheus-cpp::core)
+if(TARGET prometheus-cpp::util)
+  list(APPEND PROMETHEUS_CPP_TARGETS prometheus-cpp::util)
+endif()
+target_link_libraries(opentelemetry_exporter_prometheus
+                      PUBLIC opentelemetry_metrics ${PROMETHEUS_CPP_TARGETS})
 
 if(OPENTELEMETRY_INSTALL)
   install(

